resources/locale/en.yml

```diff
@@ -1,5 +1,8 @@
 fof-user-directory:
     forum:
+        header:
+            back_to_user_directory_tooltip: Back to User Directory
```

Comment:
See the above comment, in which case this is not required. Plus I didn't see this translation actually used when testing locally anyway..

Reply:
@imorland This text is used when you enter the User directory and then another page
![image](https://user-images.githubusercontent.com/25438601/110147968-8b671d80-7ddc-11eb-92da-0756ff4042f5.png)

Reply:
🤦 of course it is. Sorry, it's been a loooooong day.